hoomd/mpcd/__init__.py

```diff
@@ -30,57 +30,36 @@
+    * :class:`~hoomd.mpcd.collide.AndersenThermostat`
 
 Solute particles can be coupled to the solvent during the collision step. This
 is particularly useful for soft materials like polymers. Standard molecular
 dynamics integration can be applied to the solute. Coupling to the MPCD
 solvent introduces both hydrodynamic interactions and a heat bath that acts as
-a thermostat. In the future, fluid-solid coupling will also be introduced during
-the streaming step to couple hard particles and boundaries.
+a thermostat.
 
-Details of this implementation of the MPCD algorithm for HOOMD-blue can be found
+The solvent can additionally be coupled to solid boundaries (with no-slip or
+slip boundary conditions) during the streaming step.
+
+Details of HOOMD-blue's implementation of the MPCD algorithm can be found
 in `M. P. Howard et al. (2018) <https://doi.org/10.1016/j.cpc.2018.04.009>`_.
+Note, though, that continued improvements to the code may cause some deviations.
 
 .. rubric:: Getting started
 
 MPCD is intended to be used as an add-on to the standard MD methods in
-:py:mod:`hoomd.md`. To get started, take the following steps:
-
-    1. Initialize any solute particles using standard methods (`Simulation`).
-    2. Initialize the MPCD solvent particles using one of the methods in
-       :py:mod:`.mpcd.init`. Additional details on how to manipulate the solvent
-       particle data can be found in :py:mod:`.mpcd.data`.
-    3. Create an MPCD :py:obj:`~hoomd.mpcd.integrator`.
-    4. Choose the appropriate streaming method from :py:mod:`.mpcd.stream`.
-    5. Choose the appropriate collision rule from :py:mod:`.mpcd.collide`, and set
-       the collision rule parameters.
-    6. Setup an MD integrator and any interactions between solute particles.
-    7. Optionally, configure the sorting frequency to improve performance (see
-       :py:obj:`update.sort`).
-    8. Run your simulation!
-
-Example script for a pure bulk SRD fluid::
-
-    import hoomd
-    hoomd.context.initialize()
-    from hoomd import mpcd
-
-    # Initialize (empty) solute in box.
-    box = hoomd.data.boxdim(L=100.)
-    hoomd.init.read_snapshot(hoomd.data.make_snapshot(N=0, box=box))
-
-    # Initialize MPCD particles and set sorting period.
-    s = mpcd.init.make_random(N=int(10*box.get_volume()), kT=1.0, seed=7)
-    s.sorter.set_period(period=25)
-
-    # Create MPCD integrator with streaming and collision methods.
-    mpcd.integrator(dt=0.1)
-    mpcd.stream.bulk(period=1)
-    mpcd.collide.srd(seed=42, period=1, angle=130., kT=1.0)
+`hoomd.md`. Getting started can look like:
 
-    hoomd.run(2000)
+    1. Initialize the solvent through `Snapshot.mpcd`. You can include any solute
+       particles in the snapshot as well.
+    2. Create the MPCD `Integrator`.
+    3. Choose the streaming method from :mod:`.mpcd.stream`.
+    4. Choose the collision rule from :mod:`.mpcd.collide`.
+    5. Setup solute particle integration methods and interactions as you normally
```

Comment:
The main restriction is that the embedded solute particles cannot be in a rigid body. (We are going to remove this restriction as part of a funded project on this code.) I think this was explicit in the old version of the docs but got dropped out in revision. I'll add it back in `mpcd.collide.CollisionMethod`, and see if this statement can also be tightened up since it sounds more restrictive than it actually is!